FindLib/etc/makefile

```diff
@@ -9,13 +10,27 @@ DEP_DIR := depends
 OPTIMIZER		=	-O0
```

Comment:
The main makefile uses `-O3`. Should this be changed to match?

Reply:
I would say yes

Reply:
done